electrum_nmc/electrum/commands.py

```diff
@@ -1952,6 +1984,7 @@ def eval_bool(x: str) -> bool:
     'trigger_txid':(None, "Broadcast the transaction when this txid reaches the specified number of confirmations"),
     'trigger_name':(None, "Broadcast the transaction when this name reaches the specified number of confirmations"),
     'options':     (None, "Options in Namecoin-Core-style dict"),
+    'commitment_only': (None, "Only return the pre-registration commitment (use if you're pre-registering a name for someone else)"),
```

Comment:
This help text is a bit misleading, since this parameter will beset by the person requesting a name, not the person paying for such a name..